### DISCUSSION TITLE
Can't connect to the charger

### DISCUSSION THREAD

**Discussion by Naveensparks** [Lamerat/OCPP-Charge-Point-Simulator#4] [General]
I have successfully deployed and my server is up and running.
I'm not sure whether i'm doing it right.

![image](https://github.com/Lamerat/OCPP-Charge-Point-Simulator/assets/47047373/35b116e8-c95c-4e3f-818c-1a8379e2abc6)

**Comment by Lamerat** [comment]
Hello,
you set correct server **address**, **port**, **protocol**?
![Scr](https://github.com/Lamerat/OCPP-Charge-Point-Simulator/assets/83166598/ca486d1d-e564-4e7d-8f0a-f4e1c2f69b1f)

**Comment by Naveensparks** [comment]
My bad, it has been resolved now. I had issues with my connection.
Thank you
